agent/action/arbitrage_result.py

```
@@ -105,114 +320,180 @@ def run(self, context: Context, argv: CustomAction.RunArg) -> bool:
+            elif sell_result:
+                # 金币读不到(画面不在出售界面/OCR失手),退回原行为但如实标注未验证
+                mfaalog.info(f"[Arbitrage] ➖ [{item_name}] 售卖链执行完毕（金币不可读，未验证）")
             else:
+                sold_fail.append(item_name)
                 mfaalog.warning(f"[Arbitrage] ❌ [{item_name}] 售卖流程中断或失败，继续尝试下一个。")
-                
-        mfaalog.info("[Arbitrage] 🎉 所有售卖派发任务执行结束！")
+
+        if sold_fail:
+            mfaalog.warning(
+                f"[Arbitrage] ⚠️ 本轮 {len(sold_fail)}/{len(targets_to_sell)} 项未能售出："
```

Ruff (0.15.21)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)